wasm-tests/tests/wasm.rs

```diff
@@ -26,9 +26,12 @@ async fn wasm_ws_transport_works() {
 	let exp = r#"{"jsonrpc":"2.0","result":"Substrate Node","id":1}"#;
 
 	tx.send(req.to_string()).await.unwrap();
-	let rp = rx.receive().await.unwrap();
+	let rp: ReceivedMessage = rx.receive().await.unwrap();
 
-	assert_eq!(exp, &rp);
+	match rp {
+		ReceivedMessage::Data(str) => assert_eq!(exp, &str),
```

Comment:
```suggestion
		ReceivedMessage::Data(s) => assert_eq!(exp, &s),
```

(nit: just because str is a type :))